app/components/forms/session-speaker-form.js

```diff
@@ -311,7 +311,9 @@ export default Component.extend(FormMixin, {
       this.set('data.speaker', this.get('data.speaker').toArray()[0]);
     } else if (this.get('isCFS')) {
       this.set('data.speaker', this.get('store').createRecord('speaker', {
-        email : this.get('data.currentUser.email'),
+        email : this.get('authManager.currentUser.email'),
```

Comment:
@k4ni5h Please fix lint errors. Before pushing the code or sending a PR please make sure that the tests are passing on your local machine. :)

Nice work 👍